nistats/reporting/glm_reporter.py

```diff
@@ -0,0 +1,561 @@
+import io
+import string
+import os
+
+from collections import OrderedDict
+
+import pandas as pd
+
+from matplotlib import pyplot as plt
+from nilearn.datasets import load_mni152_template
+from nilearn.plotting import (plot_glass_brain,
+                              plot_roi,
+                              plot_stat_map,
+                              )
+from nilearn.plotting.js_plotting_utils import HTMLDocument
+
+from nistats.reporting import (plot_contrast_matrix,
+                               plot_design_matrix,
+                               get_clusters_table,
+                               )
+from nistats.thresholding import map_threshold
+
+
+html_template_root_path = os.path.dirname(__file__)
+
+
+def make_glm_report(
+        model,
+        contrasts,
+        title='auto',
+        roi_img=None,
+        bg_img='MNI 152 Template',
+        threshold=3.09,
+        alpha=0.01,
+        cluster_threshold=None,
+        height_control='fpr',
+        min_distance=8.,
+        plot_type='stat',
+        display_mode=None,
+        cut_coords=None,
+        nb_width=1600,
```

Comment:
why `nb`?

Reply:
This is only useful if someone wants to insert the report in a Jupyter notebook. Other wise its an HTML Page, where this has no meaning.

Reply:
I wanted to make it clear the width & height are only useful for notebooks hence the `nb_`. You think it hinders more than it helps?

Reply:
indeed, for me nb = number